bin/setup

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env ruby
+require "pathname"
+require "rake"
+
+include FileUtils
+
+ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"] || "development"
+ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
+
+def cat(source, target)
```

Comment:
I feel uncomfortable calling this `cat` because it's more of a file redirect plus some handling.